async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Again, don't hard-code S3 bucket that the customer won't own. Best to use SageMaker default bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Hard coding removed